### PR TITLE
Update security policies with additional Q&A

### PR DESCRIPTION
**Pull request by jasonblais**
Based on
 - https://pre-release.mattermost.com/core/pl/d9dnkwp4iir3z8sgm4xysh9cie
 - https://pre-release.mattermost.com/core/pl/tpijzykwopdytggmd98ae8aixe

### CODE REVIEW

**Comment by jasonblais** [comment]
@lindy65 Can you help with
 - editor review (grammar, correctness etc.)
 - reviewing the new questions make sense and fit into the category listed. If you're uncertain about anything (e.g. what a specific term means), let me know. In that case the customer representitive reviewing the doc would probably have the same question.

Once you've reviewed (and I've made corrections), we should queue this for review from CH/IT/LB/CS and also let SG/SK know about the updates.

File: source/process/security.md [hunk 1]
```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
...
+1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) logged at all times and reviewed on at least a quarterly basis?
    - Yes.
 
-1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases, and external social spaces, is management approval required prior to creating all user and privileged accounts (e.g., system or security administrator)?
+1. Are all system, application and device password files encrypted using an industry standard encryption algorithm where technically feasible?
+   - Yes
+
+1. Do IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) that communicate directly with the Internet contain any personally identifiable information (PII) such as: social security numbers, credit card numbers, patient health record information, or other confidential records?
```
Comment (lindy65) [inline source/process/security.md]: Would this sentence read better as:

"For all IT systems including but not limited to servers, routers, switches, firewalls and databases, do privileged accounts (e.g., system or security administrator) that communicate directly with the Internet, contain any personally identifiable information (PII) such as: social security numbers, credit card numbers, patient health record information, or other confidential records?"

File: source/process/security.md [hunk 2]
```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
...
+   - Yes
+
+1. Is all sensitive, protected health information (PHI) and personally identifiable information (PII) protected using an industry standard encryption algorithm where technically feasible?
+   - Yes
+
+1. Are information assets classified?
    - Yes.
 
-1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) logged at all times and reviewed on at least a quarterly basis?
+1. Are security roles and responsibilities of constituents defined and documented in accordance with the organization’s information security policy?
```
Comment (lindy65) [inline source/process/security.md]: Who are "constituents"?

File: source/process/security.md [hunk 3]
```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
...
+
+### Access Control
+
+1. Is access to and maintenance of applications, systems, network components (including routers, databases, firewalls, voice communications servers, voice recording servers, voice response units (VRU) etc), operating systems, virtualization components, hypervisors, or other information objects restricted to authorized personnel only?
+   - Yes.
+
+1. Is access to and maintenance of applications, systems, network components (including routers, databases, firewalls, voice communications servers, voice recording servers, voice response units (VRU) etc), operating systems, virtualization components, hypervisors, or other information objects granted based upon need-to-know job function?
+   - Yes.
+
+1. Are unique user IDs required for all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator)?
```
Comment (lindy65) [inline source/process/security.md]: Sorry, I'm not sure what this sentence is asking.  Should it read:

"Are unique user IDs required for all IT systems including but not limited to servers, routers, switches, firewalls and databases, and are these privileged accounts (e.g., system or security administrator)?"

File: source/process/security.md [hunk 4]
```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
...
+   - Yes.
+
+1. Are passwords required to access all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator)?
```
Comment (lindy65) [inline source/process/security.md]: Same as this question...  Should it read:

"Are passwords required for all IT systems including but not limited to servers, routers, switches, firewalls and databases, and are these privileged accounts (e.g., system or security administrator)?"

File: source/process/security.md [hunk 5]
```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
...
+1. Has a Business Impact Analysis been conducted?
+   - Yes.
+
+1. Is there a formal process focused on identifying and addressing risks of disruptive incidents to the organization?
+   - Yes.
+
+1. Is there an established Business Resiliency program that has been approved by management and communicated to appropriate constituents?
+   - Yes.
+
+1. Are specific response and recovery strategies defined for the prioritized activities?
```
Comment (lindy65) [inline source/process/security.md]: What does 'prioritized activities' refer to?

File: source/process/security.md (hunk 1 above)
Comment (jasonblais) [reply source/process/security.md]: Yes! Updated

File: source/process/security.md (hunk 2 above)
Comment (jasonblais) [reply source/process/security.md]: Would "personnel" work better? (basically it's Mattermost staff)

File: source/process/security.md (hunk 3 above)
Comment (jasonblais) [reply source/process/security.md]: I actually meant

```
Are unique user IDs required for all user and privileged accounts (e.g., system or security administrator) to access all IT systems including but not limited to servers, routers, switches, firewalls and databases.
```

Let me know if that works?

File: source/process/security.md (hunk 4 above)
Comment (jasonblais) [reply source/process/security.md]: Same thing here:

```
Are passwords required for all user and privileged accounts (e.g., system or security administrator) to access all IT systems including but not limited to servers, routers, switches, firewalls and databases.
```

File: source/process/security.md (hunk 5 above)
Comment (jasonblais) [reply source/process/security.md]: Maybe we can re-word it to `Are specific response and recovery strategies defined for addressing risks of disruptive incidents to the organization.`

File: source/process/security.md (hunk 2 above)
Comment (lindy65) [reply source/process/security.md]: Is there a reason we can't put "Mattermost staff" which would explain it fully?

File: source/process/security.md (hunk 3 above)
Comment (lindy65) [reply source/process/security.md]: Yep, that works.  I've updated.

File: source/process/security.md (hunk 4 above)
Comment (lindy65) [reply source/process/security.md]: Yep, sounds good, I've updated.

File: source/process/security.md (hunk 5 above)
Comment (lindy65) [reply source/process/security.md]: Sounds good, I've updated.

File: source/process/security.md (hunk 2 above)
Comment (jasonblais) [reply source/process/security.md]: We seem to use "personnel" in other places of this doc, so thought it might be more consistent.

Comment (lindy65) [reply source/process/security.md]: Ah, ok, I'll update it to "personnel" then :)  More explanatory that "constituents" I think...

**Comment by it33** [comment]
@lfbrock I think adding the monthly update of dependencies to the release process is probably more important than in this overview--in my mind, there should be a check to catch any dependencies that are out-of-date and understand any issues 

I can't quite see what's changed, but the pandemic add is useful, so I think we're only doing good here,

**Comment by lfbrock** [comment]
@it33, it's already in the release process that dependencies are already updated every month.

**Comment by jasonblais** [comment]
Confirmed with IT that this is good to ship. Merging. Thanks all for the reviews!

**Pull request merged**